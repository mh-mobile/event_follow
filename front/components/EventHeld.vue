<template lang="pug">
.event_held
  .event_date
    | 12/24
  .event_start
    | 開催
</template>

<style lang="scss" scoped>
.event_held {
  width: 100px;
  height: 100px;
  border: 1px solid #cccccc;
  background-color: #f3f4f7;
  border-radius: 5px;
  display: flex;
  flex-direction: column;
  justify-items: center;

  .event_date {
    font-size: 2em;
    height: 50%;
    text-align: center;
  }

  .event_start {
    font-size: 1.5em;
    height: 50%;
    text-align: center;
  }
}
</style>
