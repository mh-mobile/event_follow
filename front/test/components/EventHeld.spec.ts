import { createLocalVue, shallowMount } from "@vue/test-utils"
import VueCompositionApi from "@vue/composition-api"
import EventHeld from "@/components/EventHeld.vue"

const localVue = createLocalVue()
localVue.use(VueCompositionApi)
describe("EventHeld.vue", () => {
  it("renders html correctly", () => {
    const wrapper = shallowMount(EventHeld, {
      localVue,
      propsData: {
        started_at: "2021-02-06T13:40:00.000Z"
      }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
