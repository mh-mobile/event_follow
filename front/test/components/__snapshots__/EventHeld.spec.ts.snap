// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventHeld.vue renders html correctly 1`] = `
"<div class=\\"event_held\\" started_at=\\"2021-02-06T13:40:00.000Z\\">
  <div class=\\"event_date\\">
    <div class=\\"event_month\\"></div><span>/</span>
    <div class=\\"event_day\\"></div>
  </div>
  <div class=\\"event_start\\">開催</div>
</div>"
`;
